tools/gui/plotter/plotter_backend.py

```diff
@@ -0,0 +1,62 @@
+# This should probably go with the rest of the code for the plotter tab
+import os
+
+import pandas
+import matplotlib
+matplotlib.use('tkagg')
+import matplotlib.pyplot as plt
+
+matplotlib.rc('font', **{'sans-serif' : 'Arial',
```

Comment:
Prefer using constants for configurable attributes like font type and family.
```suggestion
PLOT_FONT_PARAMETERS={'sans-serif' : 'Arial', 'family' : 'sans-serif'}
matplotlib.rc('font', **PLOT_FONT_PARAMETERS)
```